src/algorithms/postprocessing/eqOdds.jl

```diff
@@ -144,3 +150,5 @@ function MMI.predict(model::EqOddsWrapper, fitresult, Xnew)
 	end
 	return ŷ
 end
+
+MMI.target_scitype(::Type{<:EqOddsWrapper}) = AbstractVector{<:Finite{2}}
```

Comment:
You may as well add

```
MMI.input_scitype(::Type{<:EqOddsWrapper}) = Table
```

which at least restricts user to `Table`.

The other possibility here is to make the classifier type a type-parameter `M`, as in:

```
mutable struct EqOddsWrapper{M<:MLJBase.Model} <: DeterministicComposite
	grp::Symbol
	classifier::M
end
```
and then define 

```
MMI.input_scitype(::Type{<:EqOddsWrapper{M}}) where M = input_scitype(M)
MMI.target_scitype(::Type{<:EqOddsWrapper{M}}) where M = target_scitype(M)
```

Since yours is not the kind of model that we put in the registry (as a wrapper), this might be slightly more useful (eg, in automatic type checks). This is what we do, for example, with `TunedModel`. 

But I leave this up to you.

Reply:
Ok. I will try this out.

Reply:
I have tried this and it would quite in automated checks. But the user will not be able to change the classifier once specified. Though I don't really expect that it should be a big issue. User can create new wrapper for every model he wants to wrap.

Also there should be a check that the target scitype is AbstractVector{<:Finite{2}}. So instead of `MMI.target_scitype(::Type{<:EqOddsWrapper{M}}) where M = target_scitype(M)`, we can have `MMI.target_scitype(::Type{<:EqOddsWrapper{M}}) where M = AbstractVector{<:Finite{2}}
`
The MMI.input_scitype you have specified works fine.